### ISSUE TITLE
Please port fedora_active_user to use fasjson

### ISSUE THREAD

**Issue by knight-of-ni** [pypingou/fedora-active-user#23]
A recent Fedora infrastructure change has caused fedora_active_user.py to crash, due to a HTTP 500 error while polling https://admin.fedoraproject.org/accounts/.

See here for more details:
https://pagure.io/fedora-infrastructure/issue/10832

Support for the old url has been temporarily re-enabled, but the long term solution is to port fedora-active-user over to fasjson.
https://fasjson.readthedocs.io/en/latest/

**Comment by musicinmybrain** [comment]
It looks like this is really broken now:

```
ERROR:active-user:ServerError(https://admin.fedoraproject.org/accounts/json/person_by_username, 404, Not Found)
```

**Comment by gotmax23** [comment]
Does it work with --nofas?

**Comment by musicinmybrain** [comment]
> Does it work with --nofas?

Yes. Thanks for pointing that out. It’s a lot better than nothing.

**Comment by hrw** [comment]
fasjson does not report 'last_seen' field so checking FAS can be removed.
